src/components/Image/Image.st.css

```diff
@@ -16,6 +67,18 @@
   min-height: 100%;
 }
 
+.root.overrideStyleParams {
+  background-color: value(backgroundColor);
+  border: value(borderWidth) solid value(borderColor);
+  border-radius: value(borderRadius);
+  opacity: value(containerOpacity);
+}
+
+.root.overrideStyleParams .absoluteImage, 
+.root.overrideStyleParams .relativeImage {
+  opacity: value(imageOpacity);
+}
+
```

Comment:
In all the other components we put the overrideStyleParams styling at the bottom of the st.css file. 
Do we want to keep this convention?